packages/perseus/src/widgets/matrix/matrix.tsx

```diff
@@ -537,16 +533,27 @@ class Matrix extends React.Component<Props, State> implements Widget {
     }
 }
 
-const propTransform: (arg1: any) => any = (editorProps) => {
+// NOTE: Answers here in RenderProps does not contain answers from the rubric.
+// It's an empty array of the same length and depth used to render the matrix.
+type RenderProps = MatrixPublicWidgetOptions & {
+    answers: ReadonlyArray<ReadonlyArray<string>>;
+};
+
+function transform(widgetOptions: MatrixPublicWidgetOptions): RenderProps {
     // Remove answers before passing to widget
-    const blankAnswers = _(editorProps.matrixBoardSize[0]).times(function () {
-        return stringArrayOfSize(editorProps.matrixBoardSize[1]);
+    const blankAnswers = _(widgetOptions.matrixBoardSize[0]).times(function () {
+        return stringArrayOfSize(widgetOptions.matrixBoardSize[1]);
     });
-    editorProps = _.pick(editorProps, "matrixBoardSize", "prefix", "suffix");
-    return _.extend(editorProps, {
+    widgetOptions = _.pick(
+        widgetOptions,
+        "matrixBoardSize",
+        "prefix",
+        "suffix",
+    );
+    return _.extend(widgetOptions, {
         answers: blankAnswers,
     });
```

Comment:
As long as you're refactoring this, could we move away from underscore?

```suggestion
    const {matrixBoardSize, prefix, suffix} = widgetOptions;
    return {
        matrixBoardSize,
        prefix,
        suffix,
        answers: blankAnswers,
    }
```